AWSCore/Authentication/AWSCredentialsProvider.h

```diff
@@ -237,7 +238,8 @@ typedef NS_ENUM(NSInteger, AWSCognitoCredentialsProviderErrorType) {
                     identityPoolId:(NSString *)identityPoolId
                      unauthRoleArn:(nullable NSString *)unauthRoleArn
                        authRoleArn:(nullable NSString *)authRoleArn
-           identityProviderManager:(nullable id<AWSIdentityProviderManager>)identityProviderManager;
+           identityProviderManager:(nullable id<AWSIdentityProviderManager>)identityProviderManager
```

Comment:
As an SDK we do not make breaking API changes. Can you overload the current method and add `sessionDuration` as a parameter in the overloaded method instead of modifying the current method?